app/views/pdftemplates/applicationPdf.scala.html

```diff
@@ -0,0 +1,269 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.FrontendAppConfig
+@import uk.gov.hmrc.play.views.html._
+@import controllers.routes._
+@import models.ReturnSamples
+@import models.Mode
+@import models.response.FilestoreResponse
+@import views.ViewUtils._
+@import utils.Dates
+
+@(appConfig: FrontendAppConfig, c: Case, attachments: Seq[FilestoreResponse])(implicit request: Request[_], messages: Messages)
+
+
+@createLink(path: String)(implicit req: play.api.mvc.Request[_]) = @{
+    val protocol = if(req.secure) "https" else "http"
+    s"$protocol://${req.host}${routes.Assets.versioned(path)}"
+}
+
+@sectionHeading(headerId: String) = {
+    <h2 id=@headerId>@messages(headerId)</h2>
+}
+
+
+<html>
+    <head>
+        <style>
+                body {
+                    font-size: 21px;
+                    font-family: "nta", Arial, sans-serif;
+                }
+
+                table {
+                    width: 100%;
+                    font-size: 21px;
+                    line-height: 1.31579;
+                    border-collapse: collapse;
+                }
+
+                td {
+                    padding: 8px 0px 8px 0px;
+                    border-bottom: 1px solid #bfc1c3;
+                    vertical-align: top;
+                }
+
+                td:nth-child(n) {
+                    width: 65%;
+                }
+
+                .title {
+                    align-content: center;
+                    text-align: center;
+                }
+
+                table.header td:nth-child(1) {
+                    width: 10%;
+                }
+
+                table.header td {
+                    border-bottom: 0px
+                }
+        </style>
+
+    </head>
+    <body>
+        <div class="centered-content">
+            <table class="header">
+                <tr>
+                    <td><img src="@createLink("images/hmrc-logo.jpg")" width="150" height="100"></td>
+                    <td> <h1 class="title">@messages("pdf.application.title")</h1> </td>
+                </tr>
+            </table>
+
+            <p>@messages("pdf.application.submitted"): @Dates.format(c.createdDate)</p>
+            <p>@messages("pdf.application.reference"): GB@c.reference</p>
```

Comment:
I actually think the GB but was put in the confirmation page by mistake, I think we should ask Matt if it’s meant to be there (and here)

I think it’s EORIs that start with GB

Reply:
yes, we should remove `GB`

Reply:
'GB' removed from pdf and confirmation page.  I retained the use of messages to format the reference so it can be easily changed back if required.